### PR TITLE
Add OpenJDK 16ea18 (#329)

### PR DESCRIPTION
**Pull request by govi20**
This PR adds OpenJDK early access version 18 (https://download.java.net/java/early_access/jdk16/18/GPL/{binary_file})

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala [hunk 1]
```diff
@@ -162,22 +162,22 @@ class OpenJdkMigrations {
       }
 
   @ChangeSet(
-    order = "077",
-    id = "077-add_openjdk_java_16-ea-17",
+    order = "079",
+    id = "079-add_openjdk_java_16-ea-18",
     author = "eddumelendez"
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala:167]: it should be your github username

File: src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala [hunk 2]
```diff
@@ -162,22 +162,22 @@ class OpenJdkMigrations {
       }
 
   @ChangeSet(
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala:164]: please move the whole changeset to the end. next to changeset 078

**Review by eddumelendez — CHANGES_REQUESTED** [review]
@govi20 thanks for the PR. I habe added few conments.

File: src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala (hunk 1 above)
Comment (govi20) [reply src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala:167]: updated

File: src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala (hunk 2 above)
Comment (govi20) [reply src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala:164]: moved it to the end of the file.

**Review by eddumelendez — APPROVED** [review]
LGTM!

**Pull request merged**